### DISCUSSION TITLE
Using updated Franklin-Paxinos labels

### DISCUSSION THREAD

**Discussion by KacperKon** [JingyiGF/HERBS#32] [Q&A]
Hi,
I want to use the updated Franklin-Paxinos labels mapped to the Allen CCF (as described in this paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6838086/#MOESM6), but I'm still confused about what input should be provided to HERBS.

1) Segmentation/label file - as far as I understand, the authors provided the labels as a series of annotated .tiff files (here's the link: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6838086/bin/41467_2019_13057_MOESM6_ESM.zip). Is it possible to use this kind of data?
2) Volume atlas template file - because the authors mapped their labels to Allen CCF, I guess this can be the same as for the standard Allen Brain Atlas? (So, for example, the 10um version from this site: http://download.alleninstitute.org/informatics-archive/current-release/mouse_ccf/average_template/ ?)
3) Label info file - this seems to be straightforward to prepare, based on the list of regions provided by the authors: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6838086/bin/41467_2019_13057_MOESM4_ESM.xlsx.

Do you think it's possible to use this labels directly in HERBS? Or should I simply use the Allen atlas and write some separate code to find corresponding Franklin-Paxinos labels afterwards?

Thank you for the amazing work that you're doing!

**Comment by JingyiGF** [comment]
HI, The label file you mentioned contains 123 coronal cuts matching Allen mouse CCFv3 10um atlas from Z95 to Z1315 with 100um evenly Z spacing. What do you want to do with the in-between slices? For example. do you want Z96-Z100 slices have the same label as Z95? If you want to do this, it is easy, ugly and not precise and I can provide you a file in some weeks. If you want a more precise solution, you need to fill in the labels for the in-between slices. Since I have been sick for more than 2 weeks and all my work is pending/delayed, I have no enough time to look at volume interpolation before next Autumn. You can try some other softwares to fill in the labels, for example,  https://ecosystem.supervise.ly/apps/volume-interpolation. I am sure there are many other softwares to do volume interpolation. After you got labels for each individual slices from Z95 to Z1315 (no matter it is saved in tif or nii or nrrd), things will work. Good luck!

**Comment by KacperKon** [comment]
Thanks a lot for the explanation, it's much more clear now! I'll try to do the volume interpolation and will let you know how it worked.

**Reply by ms-cassafrass** [reply]
Hello! Were you able to get this to work?
